Library/Formula/sdl_ttf.rb

```diff
@@ -18,8 +18,13 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
+  patch :DATA
+
   def install
     ENV.universal_binary if build.universal?
+    inreplace "SDL_ttf.pc.in", "@prefix@", HOMEBREW_PREFIX
```

Comment:
Shouldn't be `HOMEBREW_PREFIX` if we have to set this. Should be just `prefix`.

(We could also probably use an `inreplace` for the `SDL_ttf.c` change as well)